tests/api/test_account_permission_group.py

```diff
@@ -40,6 +44,11 @@ def test_permission_group_create_mutation(
     permission_manage_users,
     permission_manage_service_accounts,
 ):
+    staff_user2 = User.objects.get(pk=staff_user.pk)
+    staff_user2.id = None
+    staff_user2.email = "test@example.com"
+    staff_user2.save()
```

Comment:
A comment saying it creates a new one, could be nice

Reply:
I think it's clear and comment here is useless.

Reply:
Explicit is better than having to guess the magic happening.
okay... the code getting a user... copying it... changing it's ID (easily missable), change its email and then... save the copied user. Does it override? Oh wait no. Seems like django is doing an implicit create in background because the ID is none.

I prefer the user reading the code of someone else to avoid to having to go through all this though process and be able to go on with his life by looking at one short comment saying "Create a second user object for ensuring blablabla". Comments are there for that: making it quick to understand, not waste time reading and analyzing each line and noticing every details and magics hiding underneath of django.

Comments save time for maintaining and we trust them, we don't try to analyze each lines on by one.

Reply:
Quick `cmd+shift+f` in my VSCode reveals almost 50 usages of this pattern in our tests, I think it's pretty familiar to any Django dev.

Reply:
We need at least 2 staff users in these tests. We should introduce fixture like `staffs_users` and inside it use bulk_create.